apps/nextjs-website/react-components/types/BannerLink/BannerLink.types.ts

```diff
@@ -1,18 +1,21 @@
-import { CommonProps, CtaButtonProps, Generic } from "../common/Common.types";
+import { CommonProps, CtaButtonProps } from "../common/Common.types";
 
-type ImgProps = React.DetailedHTMLProps<
+export type ImgProps = React.DetailedHTMLProps<
   React.ImgHTMLAttributes<HTMLImageElement>,
   HTMLImageElement
 >;
 
-export interface BannerLinkProps
-  extends CommonProps,
-    BannerLinkContentProps {
+export interface BannerLinkSectionProps {
+  title: string;
+  normalText: string;
+  boldText: string;
+  link: string;
+  extraNormalText?: string;
+  icon?: React.ReactNode;
```

Comment:
```suggestion
  icon?: 'HubOutlined' | 'AutoFixHighOutlined' | 'MarkunreadMailboxOutlined' | 'CheckCircleOutlined' | 'SavingsOutlined' | 'HourglassEmptyRounded' | 'EnergySavingsLeafOutlined' | 'CloudOutlined';
```
```suggestion
  icon?: EIconProps;
```

This should either be a string, `EIconProps` or a list of valid MUI Icons the same as `FeatureItemMUIIconCodec`.